src/unwinder/i386/unwinder_i386.cpp

```diff
@@ -105,8 +105,19 @@ OOPStackUnwinderX86::VirtualUnwind(
     ENUM_CALLEE_SAVED_REGISTERS();
 #undef CALLEE_SAVED_REGISTER
 
-    ContextRecord->Esp = rd.SP - codeInfo.GetCodeManager()->GetStackParameterSize(&codeInfo);
-    ContextRecord->ResumeEsp = rd.SP;
+    SIZE_T paramSize = codeInfo.GetCodeManager()->GetStackParameterSize(&codeInfo);
+    SIZE_T paddingSize = 0;
+
+#ifdef UNIX_X86_ABI
+    // On UNIX_X86_ABI, function call may have stack alignment padding.
+    if (paramSize % 16 != 0)
```

Comment:
@janvorli I agree that it is a better approach to enable FEATURE_FIXED_OUT_ARGS (may be with global calling convention change from STDCALL to CDECL?). 

I once tried that, but unfortunately it requires non-trivial amount of works.

I proposed this PR as a short term workaround (I used ``UNIX_X86_ABI`` explicitly here to annotate that).